torch_np/tests/numpy_tests/core/test_multiarray.py

```diff
@@ -4179,63 +4179,48 @@ def test_kwargs(self):
             np.putmask(a=x, values=[-1, -2],  mask=[0, 1])
 
 
-@pytest.mark.xfail(reason='TODO')
 class TestTake:
     def tst_basic(self, x):
         ind = list(range(x.shape[0]))
-        assert_array_equal(x.take(ind, axis=0), x)
+        assert_array_equal(np.take(x, ind, axis=0), x)
```

Comment:
Here and elsewhere in this test module: is there at least a smoke test to probe both `np.take` and `x.take`? Maybe we don't need more than a smoke test

Reply:
Yeah it felt kinda weird to implement a smoke test, as https://github.com/data-apis/array-api-tests/pull/173 suffices for now (obv not run on our CI but was used locally) but isn't fully comprehensive of NumPy's semantics anyway, so it'd be redundant after importing more of NumPy's own tests anywho. I treated the unxfail'd tests as smoking too.